package.json

```diff
@@ -10,16 +10,16 @@
   "license": "MIT",
   "dependencies": {
     "@microsoft.azure/autorest-extension-base": "1.0.13",
-    "@ts-common/azure-openapi-markdown": "^0.3.3",
+    "@ts-common/azure-openapi-markdown": "^0.4.0",
     "@ts-common/commonmark-to-markdown": "^1.1.3",
-    "@ts-common/iterator": "0.0.38",
-    "@ts-common/json": "0.0.19",
-    "@ts-common/json-parser": "0.2.2",
-    "@ts-common/property-set": "0.0.8",
-    "@ts-common/source-map": "0.2.11",
-    "@ts-common/string-map": "0.1.0",
-    "@ts-common/tuple": "0.0.0",
-    "z-schema": "^3.24.1",
+    "@ts-common/iterator": "^0.0.38",
+    "@ts-common/json": "^0.0.20",
+    "@ts-common/json-parser": "^0.2.3",
+    "@ts-common/property-set": "^0.0.8",
+    "@ts-common/source-map": "^0.2.12",
+    "@ts-common/string-map": "^0.1.1",
+    "@ts-common/tuple": "^0.0.0",
+    "@ts-common/virtual-fs": "^0.0.1",
```

Comment:
All this bumping is not nice! It means a load of code bypasses review and is terrible for readability/maintainability

Reply:
yea kind of agree with this part . we kind of need to trust that all these changes won't introduce any regressions since that code probably hasn't been reviewed and we don't know how well is tested and it doesnt have usage to be validated in the field before we use it

---
In reply to: [224907175](https://github.com/Azure/oav/pull/339#discussion_r224907175) [](ancestors = 224907175)

Reply:
I would suggest for those packages either to switch to libraries validated in the field ( i.e. with high usage) or aggregate all of them in one repo 

---
In reply to: [224915262](https://github.com/Azure/oav/pull/339#discussion_r224915262) [](ancestors = 224915262,224907175)

Reply:
In my experience, it's other way around. It's really really hard to change anything in different `utils` folders. It's became unmanageable if we keep utilities in a project. We definitely need componentization where we can isolate and improve small pieces of code. The test coverage is also much better (usually 100%) compare to OAV. Also, we would like to share this code.

I added `^`, so after this PR, version bump is not required until breaking changes.

In the same time, I agree that I need to involve people to review utility libraries as well.

Reply:
@vladbarosan we can create a package that will combine all (most) of these `ts-common` libraries.

Reply:
a bunch of utilities were very recently added though, not sure what's meant by "there were no significant changes in utilities"